src/SortableList.js

```diff
@@ -15,7 +15,7 @@ uniqueRowKey.id = 0
 
 export default class SortableList extends Component {
   static propTypes = {
-    data: PropTypes.object.isRequired,
+    data: PropTypes.oneOfType([PropTypes.array.isRequired, PropTypes.object.isRequired]), 
```

Comment:
could you change to `PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired`

Reply:
of course, I can do it. Thanks!